Tests/TestHelpers.swift

```diff
@@ -171,6 +181,24 @@ extension GitHubUserContent: TargetType {
     }
 }
 
+// MARK: - Upload Multipart Helpers
+
+extension HTTPBin {
+    static func createMultipartFormData() -> [MultipartFormData] {
```

Comment:
nitpick: we could change the name to include test in it, e.g. `createTestMultipartFormData()`